crypto/ec/ec_key.c

```diff
@@ -119,9 +119,11 @@ EC_KEY *EC_KEY_copy(EC_KEY *dest, const EC_KEY *src)
                     return NULL;
             }
             if (!BN_copy(dest->priv_key, src->priv_key))
+                BN_free(dest->priv_key);
```

Comment:
Thank you. I see. So is the public API function still responsible for freeing the memory before it assigns the allocated memory to the application?